File: src/main/java/util/Dbutil.java
package util;

import java.sql.Connection;
import java.sql.DriverManager;

public class Dbutil {
    private final String dbUrl = "jdbc:mysql://192.168.110.253:3306/library";
    private final String dbUser = "root";
    private final String dbPass = "Lxcisasb1112138";

    public Connection getConnection() throws Exception {
        String jdbc = "com.mysql.cj.jdbc.Driver";
        Class.forName(jdbc);
        return DriverManager.getConnection(dbUrl, dbUser, dbPass);
    }

    public void close(Connection conn) throws Exception {
        if (conn != null) {
            conn.close();
        }
    }
}

File: src/main/java/util/Dbutil.kt
package util

import java.sql.Connection
import java.sql.DriverManager

class Dbutil {
    private val dbUrl = "jdbc:mysql://localhost:3306/library"
    private val dbUser = "root"
    private val dbPass = "Lxcisasb181107"
    private val jdbc = "com.mysql.cj.jdbc.Driver"

    @get:Throws(Exception::class)
    val connection: Connection
        get() {
            Class.forName(jdbc)
            return DriverManager.getConnection(dbUrl, dbUser, dbPass)
        }

    @Throws(Exception::class)
    fun close(conn: Connection?) {
        conn?.close()
    }
}